internal/callgraph/model/model.go

```diff
@@ -0,0 +1,143 @@
+package model
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strings"
+)
+
+const CURRENT_VERSION = "5"
+
+type Edge struct {
+	Parent   *Node
+	CallLine int
+}
+
+type Node struct {
+	Filename      string
+	Name          string
+	Symbol        string
+	IsLibraryNode bool
+	LineStart     int
+	LineEnd       int
+	Parents       []Edge
```

Comment:
So is the callgraph "backwards"? I think the naming is a bit unconventional so some more documentation could be nice.

Reply:
We use a reversed graph as it increases performance in the search phase.